examples/customize-rows/index.js

```diff
@@ -0,0 +1,95 @@
+/*
+  See another example of how to use `customRowRender` at
+  https://github.com/Skn0tt/mui-datatables-responsive-demo
+  https://mui-datatables-responsive-demo.skn0tt.now.sh
+*/
+
+import React from 'react';
+import MuiDatatable from "mui-datatables";
```

Comment:
Missing a few things here. This line should be `import MUIDataTable from "../../src/";` and `MUIDataTable` should be used below instead of `MuiDatatable`. Also, we'll need the `import ReactDOM from "react-dom";` dependency as well.

Reply:
Well, I should have seen that myself 😅
I guess I'm used to Typescript too much...

Fixed in 925366855e065ce211d24f0d9f11b1842b46a622.